src/popup.html

```diff
@@ -65,5 +74,8 @@
     </div>
     <script src="jquery/jquery-3.2.1.min.js"></script>
     <script src="script.js"></script>
+    <script type="text/javascript">
+        
```

Comment:
Why this ?

Reply:
@gabru-md , i tested the code in the html file itself and forgot to remove it.Will delete it .